src/main.rs

```diff
@@ -227,7 +227,7 @@ impl Kopium {
         // print doc strings if requested in arguments
         if self.docs {
             if let Some(d) = doc {
-                println!("{}/// {}", indent, d);
+                println!("{}/// {}", indent, d.replace("\n", &format!("\n{}///", indent)));
```

Comment:
Probably need a space here after the ///? Normally things don't open with a space after newlines.

Reply:
Funny you should mention it - I first tried it with a space, but the generated code looked like `///  post-\n` - one space too many. But it looks like it's cert-manager's own CRDs that include spaces after the newline. I guess a second space can't hurt, will fix.